corehq/apps/app_manager/models.py

```diff
@@ -1042,19 +1047,19 @@ def active_actions(self):
             if self.requires == 'none':
                 action_types = (
                     'open_case', 'update_case', 'close_case', 'subcases',
-                    'usercase_update', 'usercase_preload',
+                    'usercase_update', 'usercase_preload', 'usercase_subcases',
                 )
             elif self.requires == 'case':
                 action_types = (
                     'update_case', 'close_case', 'case_preload', 'subcases',
-                    'usercase_update', 'usercase_preload',
+                    'usercase_update', 'usercase_preload', 'usercase_subcases',
```

Comment:
Yes. (And it would even cooler if "case_preload" was "preload_case".) "subcases" is a little different from the other actions in that it's a list of "open_subcase" actions instead of just a single action. ... I guess calling it "open_subcases" might make sense ... except it could have only one item, and then "open_subcases" could also be misleading. Reminds me of the "[two hard things][1]" in programming.

  [1]: http://martinfowler.com/bliki/TwoHardThings.html